corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -131,7 +131,7 @@ <h1 class="page-title"><%- title %></h1>
   <div tabindex="0" id="form-menu-dropdown">
      <i class="fa fa-bars dropdown-toggle" data-toggle="dropdown"></i>
      <ul class="dropdown-menu dropdown-menu-right noprint-sub-container">
-       <li tabindex="0" class="dropdown-header" id="print-header">{% trans "Print" %} <i class="fa fa-print dropdown-icon"></i></li>
```

Comment:
didn't like the icon?

Reply:
(I liked it, just would have put it to the left like we do in the left-side menu)

Reply:
I'm open to the icon, Cody thought it looked lonely haha-- figured it's easy enough to add in later (also yes to left side just then would need to fiddle with spacing margins a bit in the `.dropdown-header`